recipes/dionysus/meta.yml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dionysus" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "78c24da06fd273f0103ad4cb86331f5f4518d7f0445e2cecd0e325ca1a2d011f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
```

Comment:
Could you use 2-space indentation here, to be consistent with the rest of the files / the rest of conda-forge? And in the rest of the file.